# HG changeset patch
# User Stuart Rackham <srackham@methods.co.nz>
# Date 1228210654 -46800
# Node ID baf9c1979fb15ccd412e3ef5f8387493e2640489
# Parent  f3c1f4c71a3b77d57e4fa7c917e2df1b1713e9b6
FIXED: Text like &...; was sometimes mistaken for an entity reference -- tightened up entity reference matching.

diff -r f3c1f4c71a3b -r baf9c1979fb1 asciidoc.conf
--- a/asciidoc.conf	Sun Nov 30 10:26:58 2008 +1300
+++ b/asciidoc.conf	Tue Dec 02 22:37:34 2008 +1300
@@ -134,8 +134,8 @@ monospacedwords=
 \\&lt;\==&lt;=
 
 # Arbitrary entity references.
-(?<!\\)&amp;([#a-zA-Z]\S*?;)=&\1
-\\(&amp;[#a-zA-Z]\S*?;)=\1
+(?<!\\)&amp;([:_#a-zA-Z][:_.\-\w]*?;)=&\1
+\\(&amp;[:_#a-zA-Z][:_.\-\w]*?;)=\1
 
 #-----------
 # Paragraphs
